api/proto/printable.proto

```diff
@@ -16,7 +16,14 @@ message PaymentRequest {
     string memo = 3;
 }
 
+message TransferPayload {
```

Comment:
Some documentation would be helpful, both for the message and for each of the fields.